outputs.tf

```diff
@@ -13,3 +13,11 @@ output "virtual_network_resource_ids" {
   value       = local.virtual_network_resource_ids
   description = "A map of virtual network resource ids, keyed by the var.virtual_networks input map. Only populated if the virtualnetwork submodule is enabled."
 }
+
+output "management_group_subscription_association_id" {
+  value       = var.subscription_alias_enabled || var.subscription_management_group_association_enabled ? module.subscription[0].management_group_subscription_association_id : null
```

Comment:
What's the thinking behind this ternary here?

Do we want to output this if `var.subscription_management_group_association_enabled` is true? Why would we output this if only `var.subscription_alias_enabled` was true?

Reply:
Hey @matt-FFFFFF, thanks for the review! I had actually taken this from the call to the `subscription` module but just realised that it's unnecessary, and we can simplify the check to `var.subscription_management_group_association_enabled` being set. Good spot, fixed and pushed up! 👍